core/commands/refs.go

```diff
@@ -313,6 +313,8 @@ func (rw *RefWriter) WriteEdge(from, to key.Key, linkname string) error {
 
 	var s string
 	switch {
+	case rw.PrintFmt == "<dst>":
```

Comment:
I don't think this case is actually needed, the next case will do what youre doing here already